Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -5542,6 +5956,261 @@ def create_gql_client(url_suffix="identity-protection/combined/graphql/v1"):
+            raise DemistoException(
+                'CS Falcon CSPM returned an error.\n'
+                'Code: 500\n'
+                f'Message: {dict_safe_get(raw_response, ["errors", 0, "message"])}\n'
+                'Perhaps the policy IDs are invalid?'
+            )
+        for error in errors:
+            if error.get('code') in (400,):
+                return_warning(
+                    'CS Falcon CSPM returned an error.\n'
+                    f'Code:  {error.get("code")}\n'
+                    f'Message: {error.get("message")}\n'
+                )
+
+    if resources := raw_response.get('resources', []):
+        human_readable = tableToMarkdown('CSPM Policy Details:', resources,
+                                         headers=['ID', 'description',
+                                                  'policy_statement', 'policy_remediation', 'cloud_service_subtype',
+                                                  'cloud_platform_type', 'cloud_service_type', 'default_severity',
+                                                  'policy_type', 'tactic', 'technique'],
+                                         headerTransform=string_to_table_header)
+        return CommandResults(readable_output=human_readable,
+                              outputs=resources,
+                              outputs_key_field='ID',
+                              outputs_prefix='CrowdStrike.CSPMPolicy',
+                              raw_response=raw_response)
+    return CommandResults(readable_output='No policy details were found for the given policy IDs.')
+
+
+def cspm_list_service_policy_settings_request(policy_id: str, cloud_platform: str, service: str) -> dict[str, Any]:
+    """Do an API call to retrieve the policy settings.
+
+    Args:
+        policy_id (str): The policy ID.
+        cloud_platform (str): The cloud platform to filter by.
+        service (str): The service type to filter by.
+
+    Returns:
+        dict[str, Any]: The raw response of the API.
+    """
+    query_params: dict[str, Any] = assign_params(service=service)
+    if policy_id:
+        query_params['policy-id'] = policy_id
+    if cloud_platform:
+        query_params['cloud-platform'] = cloud_platform
+    return http_request(method='GET', url_suffix='/settings/entities/policy/v1', params=query_params,
+                        status_code=[207])
+
+
+def cs_falcon_cspm_list_service_policy_settings_command(args: dict[str, Any]) -> CommandResults:
+    """Command to list service policy settings.
+
+    Args:
+        args (dict[str, Any]): The arguments of the command.
+
+    Returns:
+        CommandResults: The command results object.
+    """
+    policy_id = args.get('policy_id', '')
+    cloud_platform = args.get('cloud_platform', '')
+    service = args.get('service', '')
+    limit = arg_to_number(args.get('limit')) or 50
+    raw_response = cspm_list_service_policy_settings_request(policy_id=policy_id, cloud_platform=cloud_platform,
```

Comment:
```suggestion

        raw_response = cspm_list_service_policy_settings_request(policy_id=policy_id, cloud_platform=cloud_platform,
```
vertical spacing to make it easier for future readers - "we finished the assignment area, now we're making requests"